src/components/AddPlaylist/AddPlaylist.tsx

```diff
@@ -29,8 +29,13 @@ const AddPlaylist = () => {
     };
 
     const createPlaylist = async () => {
-        const promise = await dispatch(playlistsAsyncActions.createPlaylist(name));
-        history.push(`playlist/${promise.payload}`)
+        try {
+            const promise = await dispatch(playlistsAsyncActions.createPlaylist(name));
+            history.push(`playlist/${promise.payload}`)
+        } catch (error) {
+            return
```

Comment:
is this for future error handling? untill then maybe set the helpertext with some generic error? or alert?